plot_app/plotted_tables.py

```diff
@@ -58,7 +58,7 @@ def get_heading_html(ulog, px4_ulog, db_data, link_to_3d_page,
         sys_name = escape(ulog.msg_info_dict['sys_name']) + ' '
 
     if link_to_3d_page is not None and \
-        any(elem.name == 'vehicle_global_position' for elem in ulog.data_list):
+        any(elem.name == 'vehicle_global_position' or elem.name == 'vehicle_gps_position' for elem in ulog.data_list):
```

Comment:
You can just replace it:
```suggestion
        any(elem.name == 'vehicle_gps_position' for elem in ulog.data_list):
```